socorro/lib/search_common.py

```diff
@@ -310,6 +321,42 @@ def fix_hang_type_parameter(parameters):
                 hang_type.value = new_values
                 hang_type.data_type = 'int'
 
+    @staticmethod
+    def fix_version_parameter(parameters):
+        """Correct the version parameter.
+
+        If version is something finishing with a 'b' and operator is
+        'has terms', replace the operator with 'starts with' to query all
+        beta versions.
+        """
+        if 'version' in parameters:
```

Comment:
shouldn't that be `if parameters.get('version'):` because none of the stuff below matters if it's there but empty.